docs/static/filebeat_modules/system/syslog/pipeline.conf

```diff
@@ -0,0 +1,27 @@
+input {
```

Comment:
Seeing `_grokparsefailure` tags in the events sent for https://github.com/elastic/beats/blob/master/filebeat/module/system/syslog/test/darwin-syslog-sample.log and https://github.com/elastic/beats/blob/master/filebeat/module/system/syslog/test/darwin-syslog.log

The document that results looks like this:

`````
{
  "_index": "filebeat-2017.03.21",
  "_type": "log",
  "_id": "AVryyDoMOVKILbq7A5JF",
  "_score": null,
  "_source": {
    "@timestamp": "2017-03-21T21:31:27.359Z",
    "offset": 13381,
    "@version": "1",
    "beat": {
      "hostname": "Rhodas-MacBook-Pro.local",
      "name": "Rhodas-MacBook-Pro.local",
      "version": "5.3.0"
    },
    "input_type": "log",
    "host": "Rhodas-MacBook-Pro.local",
    "source": "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/filebeat-5.3.0-darwin-x86_64/log_examples/system_syslog/darwin-syslog.log",
    "message": "\t\t\t\tlocked=NO",
    "type": "log",
    "tags": [
      "beats_input_codec_multiline_applied",
      "_grokparsefailure"
    ]
  },
  "fields": {
    "@timestamp": [
      1490131887359
    ]
  },
  "sort": [
    1490131887359
  ]
}
`````